src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1287,4 +1287,13 @@
   <data name="PreferTypedStringBuilderAppendOverloadsRemoveToString" xml:space="preserve">
     <value>Remove the ToString call</value>
   </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderDescription" xml:space="preserve">
+    <value>StringBuilder.Append(char) is more efficient than StringBuilder.Append(string) when the string is a single character. When calling Append with a constant, prefer using a constant char rather than a constant string containing one character.</value>
+  </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderMessage" xml:space="preserve">
+    <value>Use StringBuilder.Append(char) instead of StringBuilder.Append(string) when the input is a constant unit string.</value>
```

Comment:
Suggestion: `StringBuilder.Append(char)` -> `'StringBuilder.Append(char)'` and `StringBuilder.Append(string)` -> `'StringBuilder.Append(string)'`